collections/individual/index.php

```diff
@@ -259,6 +260,18 @@ function initializeMap(){
 					position: mLatLng,
 					map: map
 				});
+         }
+         function leafletInit() {
+				var mLatLng = [<?php echo $occArr['decimallatitude'].",".$occArr['decimallongitude']; ?>];
+            map = new LeafletMap("map_canvas", {center: mLatLng, zoom: 8});
+            const marker = L.marker(mLatLng).addTo(map.mapLayer);
+         }
+			function initializeMap(){
+            if("<?php echo $LEAFLET?>") {
```

Comment:
Sorry. Mine was a syntax question. I was wondering why:
`if(<?php echo $LEAFLET?>)` wasn't sufficient instead of `if("<?php echo $LEAFLET?>")` with the doublequotes. But agreed about `!empty`.